tensorflow/python/keras/layers/preprocessing/preprocessing_stage.py

```diff
@@ -94,3 +96,170 @@ def map_fn(x):
                                              reset_state=reset_state)
 
 
+class FunctionalPreprocessingStage(base_preprocessing_layer.PreprocessingLayer,
+                                   functional.Functional):
+  """A functional preprocessing stage.
+
+  This preprocessing stage wraps a graph of preprocessing layers into a
+  Functional-like object that enables you to `adapt()` the whole graph via
+  a single `adapt()` call on the preprocessing stage.
+
+  Preprocessing stage is not a complete model, so it cannot be called with
+  `fit()`. However, it is possible to add regular layers that may be trainable
+  to a preprocessing stage.
+
+  A functional preprocessing stage is created in the same way as `Functional`
+  models. A stage can be instantiated by passing two arguments to
+  `__init__`. The first argument is the `keras.Input` Tensors that represent
+  the inputs to the stage. The second argument specifies the output
+  tensors that represent the outputs of this stage. Both arguments can be a
+  nested structure of tensors.
+
+  Example:
+  ```python
+  from tensorflow.keras.layers.experimental import preprocessing
```

Comment:
You can't import like this.

Please use the full symbol inside the doctest. 

`tf.keras.layers.experimental.preprocessing.Normalization`